src/test/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/testpluggables/TestAnnotation.java

```diff
@@ -0,0 +1,6 @@
+package org.broadinstitute.hellbender.cmdline.GATKPlugin.testpluggables;
+
+import org.broadinstitute.hellbender.tools.walkers.annotator.Annotation;
+
```

Comment:
Add a comment explaining that these are just for testing purposes and maybe even a link to the exact test so nobody is confused in the future, as these are discovered by reflection and its not easy to track down their purpose.

Reply:
good idea, done